src/components/Header.js

```diff
@@ -9,7 +10,7 @@ export default function Header({todosAmount}) {
             </div>
 
             <SummaryPanel todosAmount={todosAmount}/>
-            
+            <LoginPanel></LoginPanel>
```

Comment:
`<LoginPanel></LoginPanel>` can be rewritten as `<LoginPanel/>` since the `LoginPanel` component does not have any children.